src/CodeGen_LLVM.cpp

```diff
@@ -1182,20 +1173,9 @@ void CodeGen_LLVM::optimize_module() {
                     asan_options, use_global_gc,
                     use_odr_indicator, destructor_kind));
             });
-#elif LLVM_VERSION >= 120
```

Comment:
This looks like you deleted an llvm 12 block

Reply:
No, I think it's just that the >=120 block was very similar to the else block here (just an extra arg to the lambda passed to registerPipelineStartEPCallback)